feat: myPage 페이지 마크업 및 스타일링

gmlyb1 opened the pull request:
### 제목 : [feat: myPage 페이지 마크업 및 스타일링]

### PR 유형

- [ ] 새로운 기능 추가
- [ ] 버그 수정
- [x] CSS 등 사용자 UI 디자인 변경
- [ ] 코드에 영향을 주지 않는 변경사항(오타 수정, 탭 사이즈 변경, 변수명 변경)
- [ ] 코드 리팩토링
- [ ] 주석 추가 및 수정
- [ ] 문서 수정
- [ ] 테스트 추가, 테스트 리팩토링
- [ ] 빌드 부분 혹은 패키지 매니저 수정
- [ ] 파일 혹은 폴더명 수정
- [ ] 파일 혹은 폴더 삭제

### PR 체크리스트

<!-- PR이 다음 요구 사항을 충족하는지 확인하세요. -->

- [ ] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [ ] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

### 🔎 작업 내용

<!-- 어떤 작업을 하셨는지 상세하게 작성해주세요-->

- 마이페이지 마크업 및 스타일링
  <br />

### 🔧 앞으로의 과제

- 내일 할 일을 적어주세요
  <br />

### 이슈

<!-- 이슈 키워드와 함께 #을 입력한 후 이슈 번호를 선택해주세요. -->
<!-- 에시 : resolves #1 -->

resolves #54

gmlyb1 committed:
마이 페이지 마크업
- 마크업 및 디자인

gmlyb1 committed:
Merge branch 'develop' into feature/myPage

devHyeon99 commented on src/pages/Mypage.jsx:
```diff
@@ -1,9 +1,92 @@
+      <div className="w-[400px] h-[730px] px-5 bg-blue-10 flex flex-col justify-center items-center gap-[25px]">
+        
+        {/* 계정 섹션 */}
+        <div className="w-full flex flex-col gap-4"> 
```
section 태그로 변경 아래 section은 div 태그로 변경

devHyeon99 commented on src/pages/Mypage.jsx:
```diff
@@ -1,9 +1,92 @@
+            <Link 
+              to="#" // 추후 지정
+              className="absolute right-6 cursor-pointer"
+              aria-label="이동"
```
정확한 레이블 명칭 사용

devHyeon99 commented on src/pages/Mypage.jsx:
```diff
@@ -1,9 +1,92 @@
+          <Link 
+            to="#" // 추후 지정 
+            className="absolute right-6 cursor-pointer"
+            aria-label="이동"
```
정확한 레이블 명칭 사용

devHyeon99 commented on src/pages/Mypage.jsx:
```diff
@@ -1,9 +1,92 @@
+            </div>
+            <Link 
+              to="#" // 추후 지정
+              className="absolute right-6 cursor-pointer"
```
absoulte 스타일링 삭제

devHyeon99 commented on src/pages/Mypage.jsx:
```diff
@@ -1,9 +1,92 @@
+        {/* 계정 섹션 */}
+        <div className="w-full flex flex-col gap-4"> 
+          <h2 className="text-gray-450 text-lg font-semibold">계정</h2>
+          <section className="w-full h-[100px] relative flex items-center justify-between p-6 bg-white rounded-[10px] shadow">
```
relative 스타일링 삭제

gmlyb1 committed:
MyPage
- aria-hidden 값 디테일 하게 수정
- 필요없는 고정값 수정
- 계정관리 페이지 이동 경로 설정
- 필요없는 absolute 삭제 relative로 통일
- 필요없는 relative 삭제

gmlyb1 committed:
relative 삭제
